.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2.3.4
+
+      - name: Setup Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies with yarn
+        run: yarn
+
+      - name: Make build
+        run: yarn build
```

Comment:
This is the "smoketest" action, it installs the dependencies, and tries to make a build. If this action fails we need to find out why. 😄

Reply:
## `yarn build`

```
$ next build
info  - Loaded env from C:\Users\bonde\Documents\repo\octoclairvoyant\.env
info  - Using webpack 5. Reason: future.webpack5 option enabled https://nextjs.org/docs/messages/webpack5
info  - Checking validity of types...
info  - Creating an optimized production build...
info  - Compiled successfully
info  - Collecting page data...
info  - Generating static pages (0/4)
info  - Generating static pages (1/4)
info  - Generating static pages (2/4)
info  - Generating static pages (3/4)
info  - Generating static pages (4/4)
info  - Finalizing page optimization...

Page                              Size     First Load JS
┌ ○ /                             27.9 kB         225 kB
├   /_app                         0 B             142 kB
├ ○ /404                          987 B           143 kB
├ ○ /about                        3.93 kB         201 kB
└ λ /auth/callback                5.26 kB         202 kB
+ First Load JS shared by all     142 kB
  ├ chunks/332.9b828d.js          23.9 kB
  ├ chunks/376.b988c1.js          16.6 kB
  ├ chunks/643.55e49b.js          11.4 kB
  ├ chunks/938.8ae28f.js          20.1 kB
  ├ chunks/962.937ca2.js          2.5 kB
  ├ chunks/commons.c7fd2e.js      14 kB
  ├ chunks/framework.3d87ec.js    42.6 kB
  ├ chunks/main.bb40f4.js         6.61 kB
  ├ chunks/pages/_app.3b99a7.js   2.31 kB
  ├ chunks/webpack.f481cd.js      1.7 kB
  └ css/bb9077804d5cb8c8b095.css  875 B

λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
○  (Static)  automatically rendered as static HTML (uses no initial props)
●  (SSG)     automatically generated as static HTML + JSON (uses getStaticProps)
   (ISR)     incremental static regeneration (uses revalidate in getStaticProps)

Done in 13.77s.
```

## `yarn type-check`

```
$ tsc --noEmit
Done in 6.48s.
```